magento/pwa-studio
# magento/pwa-studio#: Contributing page. Fix link

## Description

Regarding to https://github.com/magento/pwa-studio/pull/1557 a correct URL to "Code of Conduct" is

```
https://github.com/magento/pwa-studio/blob/develop/.github/CODE_OF_CONDUCT.md
```

## Related Issue





Closes #ISSUE_NUMBER.

## Verification Steps

1. Go to the https://magento.github.io/pwa-studio/technologies/contribute page.
2. Click on the "Code of Conduct" link

**Actual result:** you will be redirected to https://github.com/magento/pwa-studio/blob/master/.github/CODE_OF_CONDUCT.md

**Expected result:** you should be redirected to https://github.com/magento/pwa-studio/blob/develop/.github/CODE_OF_CONDUCT.md

## Affected pages
- https://magento.github.io/pwa-studio/technologies/contribute

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

Thank you!
